### ISSUE TITLE
[Bug] Build fails in the version 3.0.4

### ISSUE THREAD

**Issue by fabiiomiiguel** [adoxography/tailwind-scrollbar#78]
With the version `3.0.1` the nuxt dev works fine, but with the latest `3.0.4` I get these two errors.
<details>
  <summary>npm run dev</summary>
  
```
 ERROR  in ./node_modules/tailwind-scrollbar/src/helpers.js

Module parse failed: Unexpected token (12:34)         friendly-errors 08:47:28
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| // eslint-disable-next-line no-underscore-dangle
> const importDefault = mod => (mod?.__esModule ? mod.default : mod);
|
| module.exports = {
                                                      friendly-errors 08:47:28
 @ ./node_modules/tailwind-scrollbar/src/utilities.js 4:26-46
 @ ./node_modules/tailwind-scrollbar/src/index.js
 @ ./tailwind.config.cjs
 @ ./plugins/device-type.ts
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
                                                      friendly-errors 08:47:28

 ERROR  in ./node_modules/tailwind-scrollbar/src/variants.js

Module parse failed: Unexpected token (35:41)         friendly-errors 08:47:28
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  */
| const getDefaultFormat = (variant, config) => {
>   if (variant === 'hover' && flagEnabled?.(config(), 'hoverOnlyWhenSupported')) {
|     return '@media (hover: hover) and (pointer: fine) { &:hover }';
|   }
                                                      friendly-errors 08:47:28
 @ ./node_modules/tailwind-scrollbar/src/index.js 9:32-53
 @ ./tailwind.config.cjs
 @ ./plugins/device-type.ts
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi ./node_modules/eventsource-polyfill/dist/browserify-eventsource.js (webpack)-hot-middleware/client.js?reload=true&timeout=30000&ansiColors=&overlayStyles=&path=%2F__webpack_hmr%2Fclient&name=client ./.nuxt/client.js
 ```
 </details>

**Comment by fabiiomiiguel** [comment]
I created a pull request with a possible solution 😄

- https://github.com/adoxography/tailwind-scrollbar/pull/79

**Comment by social-deal** [comment]
Same problem here, prevents building nuxt 2 project now.

**Comment by adoxography** [comment]
Patched in v`3.0.5`.
